yb-voyager/src/utils/utils.go

```diff
@@ -487,3 +487,9 @@ func ReadTableNameListFromFile(filePath string) ([]string, error) {
 	}
 	return list, nil
 }
+
+func CompareWithoutQuotes(s1, s2 string) bool {
+	s1 = strings.Trim(s1, "\"'")
+	s2 = strings.Trim(s2, "\"'")
+	return strings.EqualFold(s1, s2)
```

Comment:
Looks like `EqualFold` does some sort of case-insensitive matching. Why do we need this? 
And why do we even need to remove quotes? 
Why won't a simple string match work here?

Reply:
We need to remove quotes because cached event can be quotes due to that `formatIfRequired` check and incoming event is not, and vice-versa.

Reply:
EqualFold() should not be there, two strings with same value but different case-sensitivity will be different. 
Thanks, good catch! Stupid Github copilot :)

Reply:
Regarding, quotes, it could potentially lead to some false positive conflicts. Let's say val1 = `"abc"` and val2= `abc` . They are not equal but will be deemed equal. 

Nevertheless, I think this is too rare a case to be bothered with, especially because functionally nothing will break. 
Pls just add a comment indicating why we need to match after removing quotes, and the possibility of false-positives that we are ignoring!

Reply:
@makalaaneesh i actually added a definite check. There won't be any false positives. 
Please take a look at recent commits.